### PR TITLE
RAINCATCH-1215 how to use the user interface demo apps

### PR DESCRIPTION
**Pull request by austincunningham**
## Motivation:
Need a guide for how to use the demo applications user interfaces.

## What?
- [x] mobile ui documentation 
- [x] portal ui documentation

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+The {WFM-RC-LongName} demo application consists of three components:
+
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: manage

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+== Mobile application
+
+The mobile application is used by a end user to interact with workorders that have been assigned to them.
+login screen requires username and password. For a list of users see 
+link:https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/src/modules/wfm-user/users.json[users.json],
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Should not longer link to core. Link to server repo.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+all users in the demo have a default password is `123`
+
+image::/shared/images/login.png[Login]
+
+Once logged in you will be presented with the workorders screen which will show the workorders assigned to the logged in user.
+In this example workorders are accident reports to be filled out at the seen of an accident.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Demo data has workorders that are accident reports. That's not example itself.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+image::/shared/images/Workorder-mobile.png[Workorders]
+
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::/shared/images/sidebar.png[Workorders]
+
+Return to the Workorders screen and click on an one of the workorders. The following screen will render
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Do not say that exact screen will render. Avoid adding screenshots of the workflow in the mobile.
Just mention that follow the steps that were created.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+- A mobile application for end users
+- A portal application for administrator users on a web browser
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as workorders. Workers can view their workorder via a mobile applicaion and administrators
+can managed workorders via the web portal.
+
+== Mobile application
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Why mobile application is described first. 
That's not how workflow works. 

I can do the quick demo for you to show how this should be represented. 
Basically it's:
Portal:
- Create workflow
- Create workorder
Finish workorder
Do not present here any steps itself as they may differ. 
Just mention that steps can come from workflow and be developed.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::/shared/images/sidebar.png[Workorders]
+
+Return to the Workorders screen and click on an one of the workorders. The following screen will render
+
+image::/shared/images/workorder-overview.png[overview]
+
+Click on `BEGIN WORKFLOW` to proceed with data entry in the following `Accident Report`.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Skip that part. We will need to explain that people will need to follow workflow but without specifying and explaining our "Accident flow workflow" this is just example - people may have different workflow selected that will invalidate this procedure.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+`Vehicle Inspection` form. The form asks for checks for fuel ,tires and lights.
+
+image::/shared/images/workorder-vehicleInspection.png[vehicle inspection]
+
+Once the `Vehicle Inspection` form is completed click on `CONTINUE` on the bottom of the screen to proceed to the 
+`Summary` which shows the status of the workorder.
+
+image::/shared/images/workorder-summary.png[summary]
+
+Once on the summary screen you have finished your submission . To proceed click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: All above should be gone and covered without explaining step details.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+to the Workorders screen. On the Workorders screen you will see that the workorder is now marked with a green tick to so completion.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+The login screen is as follows
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Skip login screen. This is so common that is not needed to document nor screenshoting it.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+image::/shared/images/login-portal.png[login portal]
+
+Onced logged in you will be presented with the following screen for admistration of workorders and workflows.
+
+image::/shared/images/portal.png[portal]
+
+From this screen you can create new workorders by clicking the `NEW WORKORDER` button on the right panal. Once clicked
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Present ideas gradually in the way people will need to setup them.
Workflows are the first! Presenting workorders before workflows makes no sense.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+it will render the following form for creating a new workorder.
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new workorder will appear in the list on middle panal after a refresh.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: panal

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+image::/shared/images/portal.png[portal]
+
+From this screen you can create new workorders by clicking the `NEW WORKORDER` button on the right panal. Once clicked
+it will render the following form for creating a new workorder.
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new workorder will appear in the list on middle panal after a refresh. 
+Once the workorder is created it will sync to the individual worker mobile that it was assigned to. To edit individual workorders
+click the workorder from the center panal list to get the following screen.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: `panal` again. :rofl:

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+
+image::/shared/images/portal-viewWorkorder.png[view workorder]
+
+In the top right hand side there is a trash icon to allow you to delete a workorder. The pencil icon is beside
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: IMHO that's obvious and we can skip that. 
We want to cover what workorder and workflows are and how to use them. 
Things like removing/adding/editing are self explanatory.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new workorder will appear in the list on middle panal after a refresh. 
+Once the workorder is created it will sync to the individual worker mobile that it was assigned to. To edit individual workorders
+click the workorder from the center panal list to get the following screen.
+
+image::/shared/images/portal-viewWorkorder.png[view workorder]
+
+In the top right hand side there is a trash icon to allow you to delete a workorder. The pencil icon is beside 
+this allows you to update a workorder's title,assignee and select a workflow, as with creating a workorder these changes will sync to
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: This statement may not be right (as people can modify this now). 
Let's avoid listing fields.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+image::/shared/images/portal-viewWorkorder.png[view workorder]
+
+In the top right hand side there is a trash icon to allow you to delete a workorder. The pencil icon is beside 
+this allows you to update a workorder's title,assignee and select a workflow, as with creating a workorder these changes will sync to 
+the workers mobile application after the `UPDATE WORKORDER` is clicked. In the bottom right hand side there is an circle with a plus icon , this gives access to the 
+create workorder screen. See the update Workorder screen below
+
+image::/shared/images/update-workorder.png[update workorder]
+
+The other feature available in the portal is the `Workflow` functionality this allows you to create or modify a workflow.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: "the other feature" - it's not feature. Just add headings. 

Story should be 
- Building your first workflow. 
- Creating workorder based on workflow.
- Finishing workorder on mobile

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 15]
```diff
@@ -0,0 +1,110 @@
...
+Workflows are made up of steps, we have a couple of example steps implemented in the demo application. Click Wokrflow on the
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: "Workflows are made up of steps, " => consist of ordered list of steps

Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: "we have a couple of example steps implemented in the demo application."
Users will not have those steps - they will have others. Let's explain step etc.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+
+image::/shared/images/update-workorder.png[update workorder]
+
+The other feature available in the portal is the `Workflow` functionality this allows you to create or modify a workflow.
+Workflows are made up of steps, we have a couple of example steps implemented in the demo application. Click Wokrflow on the 
+left panel to access the the workflow screens .
+
+image::/shared/images/workflows.png[workflows]
+
+To create a workflow click on the `CREATE WORKFLOW` button. This will bring you to the create workflow screen where you can add a title.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: That's too granular. Assume that people can read and understand labels. 
I will go with : 
When creating workflow you need to add set of steps from the list and specify name.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 17]
```diff
@@ -0,0 +1,110 @@
...
+Click `CREATE WORKFLOW` button to continue.
+
+image::/shared/images/create-workflow.png[create workflow]
+
+The add step screen will render and will allow you to name your step and select a one of the example step types from a dropdown.
+Click add step to submit your new step as part of your work flow. 
+
+image::/shared/images/workflow-nextstep.png[next step workflow]
+
+You have edit and delete options in the bottom right hand side of each step. Like workorder you have options to delete, update workflows in the top
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Too granular.

**Comment by wtrocki** [comment]
@austincunningham 
Overall documentation is really misleading and not informative. 
We really missing the point of explaining what is the core of our current UI.

I can show you the right application flow that will need to be documented.
Try to not include many screenshots. I think we can have just 3 screenshots overall. 
Portal: Workflows, Workorders, Mobile: Workorder

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc (hunk 3 above)
Comment (austincunningham) [reply docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Example is referring to the screen shot. I should make it clearer .

```In this example in the screen shot below, workorders are accident reports to be filled out at the seen of an accident.```

**Comment by wtrocki** [comment]
@austincunningham Reviewing now

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 18]
```diff
@@ -0,0 +1,103 @@
...
+- A cloud application for backend server functionality (No user interface)
+
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as Workorders. Workers can view their Workorder via a mobile applicaion and administrators
+can manage workorders via the web portal.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new Workflows.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Typo: appliction

Comment (wtrocki) [reply docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Use list of the functionalities. How about edit workflows etc.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 19]
```diff
@@ -0,0 +1,103 @@
...
+The Portal application is a web application. The view of workorders is restricted to administrators.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: > The Portal application is a web application. The view of workorders is restricted to administrators. 

This sentence doesn't make any sense.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 20]
```diff
@@ -0,0 +1,103 @@
...
+The demo application has demo users and demo data to show off the functionality of {WFM-RC-LongName}.
+The demo application can define business process or procedure via a custome Workflow and assign these workflows as jobs to
+individual workers as Workorders. Workers can view their Workorder via a mobile applicaion and administrators
+can manage workorders via the web portal.
+
+== Portal application
+
+The Portal appliction is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+The Portal application is a web application. The view of workorders is restricted to administrators. 
+Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Images are still there! Remove all login images.

Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: > Onced logged in you will be presented with the following screen for admistration of Workorders and Workflows.

That's not true as developers may change default page. 
Just mentiuon that there are workforders and workflows sections (which must be there) and you should start with workflows.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 21]
```diff
@@ -0,0 +1,103 @@
...
+
+image::/shared/images/portal.png[portal]
+
+=== Create Workflow
+
+Click on Workflow in the left panel. To create a workflow click on the `CREATE WORKFLOW` button. This will bring you to the create workflow screen where you can add a title.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: That's instruction. It should relay on numbers.

> Click on Workflow in the left panel.

Select workflow *on* navigation panel

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+Click `CREATE WORKFLOW` button to continue.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: There is no `Create workflow` button.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 23]
```diff
@@ -0,0 +1,103 @@
...
+
+image::/shared/images/create-workflow.png[create workflow]
+
+The add step screen will render and will allow you to name your step and select a one of the example step types from a dropdown (Developed custom steps will appear in the dropdown).
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Avoid obvious stuff: `The add step screen will render`

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 24]
```diff
@@ -0,0 +1,103 @@
...
+You have options to delete, update workflows in the top right hand side with the trash and pencil icons. You also can use the circle with the plus icon to create new workflows. This new Workflow 
+can be used to create or update workorders from the workorders screen as shown in the image below. 
+
+image::/shared/images/new-workflow.png[select new workflow]
+
+The new workflow will appear in the list in the center panel after a refresh.
+You can access existing you can access existing workflows by clicking on them in the list in the center panel. Options to edit the workflow are the same 
+as the create.
+
+The portal is the `Workflow` functionality this allows you to create or modify a workflow.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Rephrase.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 25]
```diff
@@ -0,0 +1,103 @@
...
+Workflows are made up of steps, there are a couple of example steps implemented in the demo application. Click Workflow on the 
+left panel to access the the Workflow screens .
+
+image::/shared/images/workflows.png[workflows]
+
+=== Create Workorder
+
+Workorders are individual instances of a Workflow that can be assigned to a worker.
+Click on Workorder in the left panel. Create new workorders by clicking the `NEW WORKORDER` button on the right panel. Once clicked
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: + button

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 26]
```diff
@@ -0,0 +1,103 @@
...
+it will render a form for creating a new Workorder. See example below
+
+image::/shared/images/create-workorder.png[create workorder]
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Not needed. Avoid so many screenshots.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
+
+Fill out the form and click `CREATE WORKORDER` and a new Workorder will appear in the list on middle panel after a refresh. 
+Once the Workorder is created it will sync to the individual workers account to be accessed on a mobile. To edit individual Workorders
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: assigned field worker

Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Note: Workorders that are finished cannot be reassigned.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 28]
```diff
@@ -0,0 +1,103 @@
...
+can be used to create or update workorders from the workorders screen as shown in the image below. 
+
+image::/shared/images/new-workflow.png[select new workflow]
+
+The new workflow will appear in the list in the center panel after a refresh.
+You can access existing you can access existing workflows by clicking on them in the list in the center panel. Options to edit the workflow are the same 
+as the create.
+
+The portal is the `Workflow` functionality this allows you to create or modify a workflow.
+Workflows are made up of steps, there are a couple of example steps implemented in the demo application. Click Workflow on the
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Add more info about steps. Add info about version bumps for the workflow.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+
+The new workflow will appear in the list in the center panel after a refresh.
+You can access existing you can access existing workflows by clicking on them in the list in the center panel. Options to edit the workflow are the same 
+as the create.
+
+The portal is the `Workflow` functionality this allows you to create or modify a workflow.
+Workflows are made up of steps, there are a couple of example steps implemented in the demo application. Click Workflow on the 
+left panel to access the the Workflow screens .
+
+image::/shared/images/workflows.png[workflows]
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: I do not think that's needed. Obvious content.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 30]
```diff
@@ -0,0 +1,103 @@
...
+
+image::/shared/images/new-workflow.png[select new workflow]
+
+The new workflow will appear in the list in the center panel after a refresh.
+You can access existing you can access existing workflows by clicking on them in the list in the center panel. Options to edit the workflow are the same 
+as the create.
+
+The portal is the `Workflow` functionality this allows you to create or modify a workflow.
+Workflows are made up of steps, there are a couple of example steps implemented in the demo application. Click Workflow on the 
+left panel to access the the Workflow screens .
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Let's document that step changes increment versions

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 31]
```diff
@@ -0,0 +1,103 @@
...
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new Workorder will appear in the list on middle panel after a refresh. 
+Once the Workorder is created it will sync to the individual workers account to be accessed on a mobile. To edit individual Workorders
+click the Workorder from the center panel list see an example in the .
+
+image::/shared/images/portal-viewWorkorder.png[view workorder]
+
+The pencil icon this allows you to update a workorder's title,assignee and select a workflow, as with creating a workorder these changes will sync to
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Let's mention why you cannot update workflow.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 32]
```diff
@@ -0,0 +1,103 @@
...
+
+=== Create Workorder
+
+Workorders are individual instances of a Workflow that can be assigned to a worker.
+Click on Workorder in the left panel. Create new workorders by clicking the `NEW WORKORDER` button on the right panel. Once clicked
+it will render a form for creating a new Workorder. See example below
+
+image::/shared/images/create-workorder.png[create workorder]
+
+Fill out the form and click `CREATE WORKORDER` and a new Workorder will appear in the list on middle panel after a refresh.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Add info: Workorder is created with the clone of the existing workflow. Future edits of the workflow will not affect current workorder. If you want to change current workorder flow you will need to remove it and create a new one.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 33]
```diff
@@ -0,0 +1,108 @@
...
+
+The Portal application is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+Once logged in you will be presented with sections for Workorder and Workflow.
+
+=== Portal Functionality
+**Workflow**
+
+- Create,Read, Update, Delete and List operations on Workflow
+- Manage Workflows and add steps to existing Workflows 
+- Changing existing Workflow will not effect Workorders already created _i.e. the Workflow will remain unchanged in existing Workorders._
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Suggest to remove 2 bellow as they aren't features.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 34]
```diff
@@ -0,0 +1,108 @@
...
+
+== Portal application
+
+The Portal application is used by a administrator to view , edit and assign Workorder and to create new Workflows.
+Once logged in you will be presented with sections for Workorder and Workflow.
+
+=== Portal Functionality
+**Workflow**
+
+- Create,Read, Update, Delete and List operations on Workflow
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Manage

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 35]
```diff
@@ -0,0 +1,108 @@
...
+- Manage Workflows and add steps to existing Workflows 
+- Changing existing Workflow will not effect Workorders already created _i.e. the Workflow will remain unchanged in existing Workorders._
+- Workflows will version bump when changed
+
+**Workorder**
+
+- Create,Read, Update,Delete and List operations on Workorder
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Manage

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 36]
```diff
@@ -0,0 +1,108 @@
...
+- Manage Workorders by assigning them to individuals
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: workers

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 37]
```diff
@@ -0,0 +1,108 @@
...
+- Once a Workorder is created Workflows remains static and unchangeable
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Encompass that as note bellow.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 38]
```diff
@@ -0,0 +1,108 @@
...
+- Workflows will version bump when changed
+
+**Workorder**
+
+- Create,Read, Update,Delete and List operations on Workorder
+- Manage Workorders by assigning them to individuals
+- Once a Workorder is created Workflows remains static and unchangeable   
+
+
+=== Create Workflow
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Createing

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 39]
```diff
@@ -0,0 +1,108 @@
...
+
+- 1. Select on Workflow in the navigation panel on the left. 
+- 2. To create a workflow click on the `NEW WORKFLOW` button. This will bring you to the create workflow screen where you can add a title.
+- 3. Click `CREATE WORKFLOW` button to continue.
+
+image::https://raw.githubusercontent.com/feedhenry-raincatcher/raincatcher-docs/master/docs/shared/images/create-workflow.png[create workflow]
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Externalize to VARIABLE

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 40]
```diff
@@ -0,0 +1,108 @@
...
+
+- 4. Name your step and select a step types from a dropdown (Developed custom steps will appear in the dropdown). 
+
+- 5. Click `ADD STEP` to submit your new step as part of your Workflow. 
+
+image::https://raw.githubusercontent.com/feedhenry-raincatcher/raincatcher-docs/master/docs/shared/images/workflow-nextstep.png[next step workflow]
+NOTE: The add step screen will allow you to add as many steps as your process requires.
+
+You have options to delete, update workflows in the top right hand side with the trash and pencil icons.
+Updating Workflows will bump the version of the Workflow.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: bump => increment

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 41]
```diff
@@ -0,0 +1,108 @@
...
+The pencil icon this allows you to update a workorder's title,assignee and select a workflow, as with creating a workorder these changes will sync to 
+the workers mobile application after the `UPDATE WORKORDER` is clicked. In the bottom right hand side there is an circle with a plus icon , this gives access to the 
+create workorder screen.
+
+== Mobile application
+Once logged in you will be presented with the workorders screen.
+To log out click on the menu icon in the top left hand side of the screen this will open a sidebar with options to Logout or return
+to the Workorders screen.
+
+image::https://raw.githubusercontent.com/feedhenry-raincatcher/raincatcher-docs/master/docs/shared/images/sidebar.png[Sidebar]
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Remove that.

File: docs/workforce-management-framework/topics/pro-using-demo-app.adoc [hunk 42]
```diff
@@ -0,0 +1,108 @@
...
+
+
+=== Complete Workorder
+The mobile application is used by a end user to interact with Workorders that have been assigned to them.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-demo-app.adoc]: Wrong place. Move up

File: docs/workforce-management-framework/topics/pro-using-the-demo-app.adoc [hunk 43]
```diff
@@ -0,0 +1,123 @@
...
+=== Creating a Workflow
+
+. Login as an administrator to the Portal Application
+. In the navigation panel on the left, click on *Workflow*
+. To create a new _Workflow_, click on *NEW Workflow*
+.. This will bring you to the _Create Workflow screen_
+. Add a title to the _Workflow_
+. Click *CREATE Workflow* to navigate to the _Add Step screen_
++
+image::{WFM-RC-images}create-Workflow.png[creating a Workflow]
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-using-the-demo-app.adoc:50]: That image will be broken for upstream. 
ping @austincunningham we may need to figure out some pattern here to support both. 
For the moment we will just leave it this way and sort that by website automation (copying images into the right directory.

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Render has broken images but we going to fix that by automation to not cause more problems to @jstaffor

**Pull request merged**